core/src/main/web/app/utils/basic/commonui.js

```diff
@@ -83,13 +83,15 @@
 
         var showDropdown = function() {
           var togglePosition = toggle.offset();
+          var $notebook = $('bk-notebook');
+          var notebookPosition = $notebook.offset();
```

Comment:
Yeah, we could do that.  I'm not sure if that'd be worth the complexity in this case though, as it's on user interaction.